docs/sources/panels-visualizations/visualizations/stat/index.md

```diff
@@ -21,24 +21,74 @@ weight: 100
 
 # Stat
 
-Stats show one large stat value with an optional graph sparkline. You can control the background or value color using thresholds or overrides.
-
-{{< figure src="/static/img/docs/v66/stat_panel_dark3.png" max-width="1025px" caption="Stat visualization" >}}
-
 {{% admonition type="note" %}}
 This visualization replaces the Singlestat visualization, which was deprecated in Grafana 7.0 and removed in Grafana 8.0.
 {{% /admonition %}}
 
+A stat panel visualizes your data in single values, such as the latest or current value that you are only interested in, with an optional graph sparkline. A graph sparkline, which is only available in stat panels, are small time-series graph shown in the background of the panel.
```

Comment:
We recently implemented a style rule that these are all called visualizations (they're in the Visualizations section :-)). Panels are a combination of data, a query, maybe a transformation, and a visualization. Time series, stat, canvas, etc are all types of visualizations that you can apply to your data.

```suggestion
A stat visualization displays your data in single values of interest, such as the latest or current value of a series, with an optional graph sparkline. A graph sparkline, which is only available in stat visualizations, is a small time-series graph shown in the background of each value in the visualization.
```

Reply:
That's great to know, thanks Isabel! I'll keep this in mind.